locales/messages.en.json

```diff
@@ -102,7 +102,7 @@
   "GraphCycleDetected": "Cycle detected within graph at {package_name}:",
```

Comment:
Please avoid changes to the per-language JSONs and the lcl files -- those are all automatically regenerated by the translation team based on `messages.json`.

Reply:
Done.